active/gridsyntax/GridSyntaxSpec2.md

```diff
@@ -0,0 +1,368 @@
+    static Windows.UI.Xaml.DependencyProperty MaxWidthProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty MinWidthProperty{ get; };
+};
+```
+
+## RowDefinition API
+
+```cs
+[webhosthidden]
+
+// vvv
+[contentproperty("Height")] // Sets the RowDefinition's content property to the Height property.
+// ^^^
+
+runtimeclass RowDefinition : Windows.UI.Xaml.DependencyObject
+{
+    RowDefinition();
+    Windows.UI.Xaml.GridLength Height;
+    Double MaxHeight;
+    Double MinHeight;
+    Double ActualHeight{ get; };
+    static Windows.UI.Xaml.DependencyProperty HeightProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty MaxHeightProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty MinHeightProperty{ get; };
+};
+```
+
+# Appendix
+
+## Alternatives
+
+## Syntax Considerations and Rationale
+In developing the syntax, the team considered developer familiarty and patterns as much as possible to ensure that the Grid learning curve could be decreased. One of the considerations was how similar the syntax should be to JSON, as that's a popular notation style that often addresses the same problems that Xaml does. 
+
+JSON defines their arrays as: ```"words" : ["apple", "banana", "cherry"]```
+
+Xaml defines a collection-typed property as: ```<Words="'Apple', 'Banana', 'Cherry'"/>```
+
+The syntax that we chose encloses strings individually in quotes, as JSON does, but it does so in a way that is more consistent with existing Xaml style and Xaml functions. An example of this syntax used elsewhere in Xaml is for the x:Bind syntax:
+
+```xml <TextBlock Text="{x:Bind VM.MyText, FallbackValue='Hello, world', Mode=TwoWay}" />```
+
+This uses a comma as a delimiter, but also allows strings to have commas within them. For these reasons, we chose to use the syntax in which a collection can be delimited by commas but have more complex values be encased within single quotes.
+
+
+## Release Details
+The xmlns will not be updated for this Xaml language feature addition, i.e. there will be no formal update to the Xaml language. Visual Studio, however, will have to be updated to accept the new syntax structure. This feature will be released with WinUI and not with the Xaml SDK.
+
+## Tooling/Visual Studio Considerations
+In terms of usage of this new Xaml language feature in Visual Studio, VS needs a way to recognize the current Xaml version that the developer is using. This will be likely be done by accessing metadata or accessing an attached features.xml file - the VS team is still in discussion as to which method will be used, but the overall cost is not high. 
+
+## Data and Intelligence Metrics
+### P0: Feature Key Performance Indictors
+* Syntax has increased developer/consumer satisfaction
+    * KPI: At least 80% of new apps released to Windows store after release of feature are using the new syntax, and 50% of developers express a positive reaction to this change.
```

Comment:
I think 80% is too aggressive. There's a lot of pre-existing XAML out there that doesn't use the new syntax and nobody is going to go through their existing XAML and refresh it before pushing a bugfix to Store.

Reply:
Recommend: We can leave these D&I goals out